src/Administration/Resources/app/administration/src/module/sw-mail-template/component/sw-mail-header-footer-list/index.js

```diff
@@ -26,6 +31,14 @@ Component.register('sw-mail-header-footer-list', {
         }
     },
 
+    watch: {
+        searchTerm: {
+            handler (value) {
```

Comment:
ESLint should cause an error here:

```suggestion
            handler(value) {
```